Lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -1126,6 +1124,13 @@ namespace Js
 #endif
       isByteCodeDebugMode(false)
     {
+        if (nestedCount > 0)
+        {
+            NestedArray* nestedArray = RecyclerNewWithBarrierPlusZ(m_scriptContext->GetRecycler(),
+                nestedCount*sizeof(WriteBarrierPtr<FunctionProxy>), NestedArray, nestedCount);
+            SetAuxPtr(AuxPointerType::NestedArray, nestedArray);
```

Comment:
What is the data that support this to be in the Aux structure?  Are most function "leaf" functions?